src/main/scala/microsites/MicrositeKeys.scala

```diff
@@ -134,6 +134,11 @@ trait MicrositeAutoImportSettings extends MicrositeKeys {
         s"/${micrositeBaseUrl.value}"
       else micrositeBaseUrl.value
 
+    val homepage =
```

Comment:
This is not needed, as I mentioned before, there is a helper method to cover it.

See [this example](https://github.com/47deg/sbt-microsites/blob/3a8e8acd0e452a539c59b7c07ead4189ada1ff84/src/main/scala/microsites/util/MicrositeHelper.scala#L58)